onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -2844,6 +2844,73 @@ void RegisterContribSchemas() {
         propagateElemTypeFromInputToOutput(ctx, 0, 0);
       });
 
+  ONNX_CONTRIB_OPERATOR_SCHEMA(EPContext)
+      .SetDomain(kMSDomain)
+      .SinceVersion(1)
+      .SetDoc("Onnx node for EP engine or context cache.")
+      .Attr(
+          "ep_cache_context",
+          "payload of the execution provider context if embed_mode=1, or path to the context file if embed_mode=0.",
+          AttributeProto::STRING)
+      .Attr(
+          "embed_mode",
+          "1: indicate ep_cache_context is the context content. 0: indicate ep_cache_context is the file path to the context content."
+          "The path is relative to this Onnx file",
```

Comment:
not sure if relative path support is enough.

Reply:
Those files can be generate on one machine and deployed to other agents. We need to make sure it's relative path I think.

---
In reply to: [1348216397](https://github.com/microsoft/onnxruntime/pull/17757#discussion_r1348216397) [](ancestors = 1348216397)